.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: ci
+on:
+  push:
+    branches:
+      - develop
+      - release/**
+      - hotfix/**
+      - feat/dev-245-build-binaries
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
+        python:
+          - 3.7
+    env:
+      GDC_CLIENT_ZIP: "gdc-client.zip"
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python }}
+    - name: Run Tests
+      if: matrix.os != 'windows-latest'
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox
+        tox -e py
+    - name: Package
+      run: |
+        cd bin
+        pip install virtualenv
+        . ./package
```

Comment:
This seems a little too clever, but it works in this context, so. 🤷‍♂️

Reply:
You what @singergr you really do detect the tiniest of details. Good eyes :+1: . I should really give credit to the stackoverflow answer that pointed me to exec it like this so the environment variable can be kept long enough to pass it to github action